packages/slackAlerter/.jest/setEnvVars.js

```diff
@@ -0,0 +1,3 @@
+process.env.ENV = "dev"
+process.env.AWS_SESSION_TOKEN = "76b6033a-232c-4b5c-8d92-39760202b2d8"
```

Comment:
can we add a comment to say this is a dummy value and not a real session token so we dont get confused when a security scan happens and finds it